### PR TITLE
Completed prep-javascript-dom-creation. Louisa Whitaker

### PR DESCRIPTION
**Pull request by louiewhitz**
Screen Shot 2022-03-04 at 11 49 32 AM

### CODE REVIEW

File: prep-javascript-dom-creation/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript DOM Creation</title>
+</head>
+
+<body>
+  <div id="container">
+    <div id="h1">Hello World!</div>
```
Comment (thebearingedge) [inline prep-javascript-dom-creation/index.html]: For this exercise we don't want to modify the HTML itself, but instead use JavaScript code to add a new element to the DOM. I recommend completing the dom manipulation and dom events exercises first, as our exercises are meant to be completed from bottom to top in your dashboard.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-javascript-dom-creation/index.html (hunk 1 above)
Comment (louiewhitz) [reply prep-javascript-dom-creation/index.html]: No wonder I was having trouble. I'll do that today.

**Review by thebearingedge — APPROVED** [review]

File: prep-javascript-dom-creation/index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript DOM Creation</title>
+</head>
+
+<body>
+  <div id="container">
+  </div>
+  <script>
+ var $container = document.querySelector('container');
+ console.log($container);
```
Comment (thebearingedge) [inline prep-javascript-dom-creation/index.html]: Actually i don't think this gives you what you want.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-javascript-dom-creation/index.html (hunk 2 above)
Comment (louiewhitz) [reply prep-javascript-dom-creation/index.html]: Okay, I fixed it back and I think I committed the new file? I hope so

Comment (louiewhitz) [reply prep-javascript-dom-creation/index.html]: Screen Shot 2022-03-05 at 1 49 57 PM

**Comment by louiewhitz** [comment]
Screen Shot 2022-03-05 at 2 23 06 PM

**Review by louiewhitz — COMMENTED** [review]
Screen Shot 2022-03-05 at 2 23 06 PM

**Review by thebearingedge — APPROVED** [review]
👍

**Pull request merged**